course/resources/eeb430-syllabus.md

```diff
@@ -0,0 +1,103 @@
+# EEB430H1 Theoretical Ecology [24L, 12P]
+This course focuses on theoretical ecology concepts and reproducible quantitative methods using R. Theoretical ecology uses models to explain biological phenomena such as the maintenance of biodiversity, population growth, competition, eco-evolutionary dynamics, epidemiology, spatial ecology, and species extinction. This course is designed to meet a growing demand for reproducible, openly accessible, analytically thorough, and well documented science. Students will learn to develop ecological population models, analyze data, and document their research using the R programming language. No prerequisite programming experience is required.
+
+Prerequisites: MAT 136, BIO220H1, EEB225H1, and two of EEB319H1, EEB321H1,
+EEB322H1, EEB323H1
+
+# Time
+Tue and Thu 2-4 pm.  
+
+# Lecture halls
+Tue, Ramsay Wright (RW 109).  
+South west corner of Harbord and St George, 1st floor, enter from St George street.
+
+Thu, Carr Hall (SMC 325).  
+Corner of Queen's Park East and St. Joseph Street, 3rd floor, St. Michael's College.
+
+Both lecture halls have access to individual computers for the students. To use the computer workstations, students can login with their UTORid and password. Installed programs and packages are wiped from the computers daily. Saved files will be accessible from any computer via the student's login. Students can use any of the lecture halls when there are no classes scheduled. See the [online schedules](http://lab.chass.utoronto.ca/carr.php) for available times.
+
+# Contact info
+
+## Teaching assistants
+Joel Ostblom (course coordinator), joel.ostblom@mail.utoronto.ca  
+Madeleine Bonsma, m.bonsma@mail.utoronto.ca  
+Luke Johnston, luke.johnston@mail.utoronto.ca  
+Lina Tran, lina.tran@mail.utoronto.ca  
+Elliott Sales de Andrade, esalesde@physics.utoronto.ca  
+Lindsay Coome, lindsay.coome@mail.utoronto.ca  
+
+_Please address all general course-related email communication to joel.ostblom@mail.utoronto.ca, and project specific communication to the respective TA of your group. Prefix the subject matter with "EEB430". If you do not receive a reply within 48 hours (excluding week-ends), please send a reminder. Office hours: Contact us by e-mail to schedule a time._
```

Comment:
We should discuss wether we want a fixed office hour time slot as per the suggestion from @lwjohnst86. Might be hard to fit into all of our schedules though and easier for them to just email. Or we could say that office hours are always one hour before or after class.